### PR TITLE
show pending CDN invalidations on queue page

### PR DESCRIPTION
**Pull request by syphar**
Since #1880 raised some more open questions, here a simple addition to the queue page. I'll finish up #1880 separately, same for actually querying the cloudfront API to get the invalidations. 

This is IMO the last piece missing to really enable #1552 in the CDN configuration.

### CODE REVIEW

**Comment by syphar** [comment]
This is how it looks like right now: 
grafik

**Comment by syphar** [comment]
ping @rust-lang/docs-rs 

I would love a review / feeback here. 

This is the last piece missing before we can activate the full page caching for docs.

File: src/cdn.rs [hunk 1]
```diff
@@ -133,14 +135,51 @@ pub(crate) fn invalidate_crate(config: &Config, cdn: &CdnBackend, name: &str) ->
...
+#[derive(Debug, Clone, Serialize, PartialEq, Eq)]
+pub(crate) struct CrateInvalidation {
+    pub name: String,
+    pub created: DateTime<Utc>,
+}
+
+/// Return fake active cloudfront invalidations.
+/// CloudFront invalidations can take up to 15 minutes. Until we have
+/// live queries of the invalidation status we just assume it's fine
+/// latest 20 minutes after the build.
```
Comment (jsha) [inline src/cdn.rs:147]: > we just assume it's fine latest 20 minutes after the build.

I think something is missing or typo'ed here?

**Review by jsha — APPROVED** [review]
This looks good to me, both UI and code. I have not tested locally. Thanks for working on this!

**Pull request merged**